DATA605/README.md

```diff
@@ -0,0 +1,224 @@
+  * [Choosing a project](#choosing-a-project)
+  * [Working](#working)
+  * [Pre-requisites](#pre-requisites)
+  * [Project assignment](#project-assignment)
+    + [Documentation](#documentation)
+    + [Submitting project](#submitting-project)
+  * [Example of a class project](#example-of-a-class-project)
+
+<!-- tocstop -->
+
+
+# DATA605 Class Projects
+
+- The goal of the class project is to learn cutting-edge modern big data
+  technology and write a (small) example of a system using it
+- Each class project is similar in spirit to the tutorials for various
+  technologies (e.g., Git, Docker, SQL, Mongo, Airflow, Dask) we have looked and
+  studied in classes
+
+## Choosing a project
+
+- Each student should pick one of the projects from the [signup sheet](https://docs.google.com/spreadsheets/d/1Ez5uRvOgvDMkFc9c6mI21kscTKnpiCSh4UkUh_ifLIw/edit?gid=0#gid=0)
+  - The difficulty of the project does affect the final grade, but we want to
+    give a way for everyone to select a project based on their level of computer
+    literacy
+  - You will need to give us some information including your GitHub name so we
+    can add you to the repo
+
+- The project is individual
+  - Students can discuss and help each other (they will do that even if we say
+    not to)
+  - Students should not have exactly the same project
+  - If there are no more projects left for any reason, then we will add more
+
+- The goal is to get your hands dirty and figure things out
+  - Often working is all about trying different approaches until one works out
+  - Google and ChatGPT are your friends, but don't abuse them: copy-pasting
+    without understanding doesn't help you
+  - Make sure you understand what, how, and why a piece of code does
+ 
+- On the sign-up sheet, you will find several baseline project proposals
+  suggested by us. You can either choose from the given projects or propose your
+  own project. You can improve, change, and add other technologies and components
+  to enhance your project
+- Your project choice should align with your learning goals and interests,
+  presenting an excellent opportunity to explore various technologies and enhance
+  your resume.
+
+
+- If you choose one of the projects from the sign-up sheet, fill out the
+  corresponding information promptly. If you decide to propose a modification,
+  send us an email with the desired information and we will update the sign-up
+  sheet and Google Doc for you.
+
+- Please note that you are required to finalize your project selection within
+  one week
+- The project duration is approximately four weeks, so timely selection is
+  crucial for effective planning and execution.
+
+- Your grades will be influenced by factors such as project complexity, your
+  efforts and understanding, and adherence to given project guidelines.
+
+## Working
+
+- You will work in the same way open-source developers (and specifically
+  developers on Sorrentum) contribute to a project
```

Comment:
We should replace all instances of Sorrentum -> Causify.AI

Sorrentum was the name of the open source part, but now we have merged them